pyupgrade.py

```diff
@@ -1086,6 +1101,22 @@ def visit_Call(self, node):
                 node.func.attr in SIX_CALLS
         ):
             self.call_attrs[_ast_to_offset(node)] = node
+        elif (
+                isinstance(node.func, ast.Attribute) and
+                isinstance(node.func.value, ast.Name) and
+                node.func.value.id == 'six' and
+                node.func.attr == 'b' and
+                len(node.args) == 1 and
+                isinstance(node.args[0], ast.Str)
+        ):
+            self.literal_attrs[_ast_to_offset(node)] = node
+        elif (
+                isinstance(node.func, ast.Name) and
+                node.func.id == 'b' and
+                node.func.id in self.six_from_imports and
+                isinstance(node.args[0], ast.Str)
+        ):
+            self.literal_attrs[_ast_to_offset(node)] = node
```

Comment:
any way that `is_six_attr` / `is_six_name` can be reused?

Reply:
I was able to do it